src/java/net/java/sen/SenFactory.java

```diff
@@ -125,10 +125,10 @@ private static InputStream getInputStream(String name, String dictionaryDir) thr
 
   private final String[] posIndex, conjTypeIndex, conjFormIndex;
   private final ByteBuffer costs, pos, tokens, trie;
-  
-  
+
   public static final String unknownPOS = "未知語";
-  
+  public static boolean compatibilityMode = true;
```

Comment:
And this property should not be static property. If we set it once and we call getStringTagger(), its behavior is different before.
I think we change it to instance property and we add 2nd param to each method.